src/components/badge/index.tsx

```diff
@@ -0,0 +1,11 @@
+import DotIcon from '@/assets/icons/DotIcon';
+
+const Badge = ({ children }: React.PropsWithChildren) => {
+  return (
+    <div className='flex w-fit items-center gap-6 rounded-full bg-[#FDF4D3] px-8 py-4 overflow-ellipsis whitespace-nowrap'>
+      <DotIcon />
+      <span className='text-xs text-[#764d4d]'>{children}</span>
```

Comment:
```suggestion
      <span className='text-xs text-[#764d4d]'>{children}</span>
```
```suggestion
      <span className='text-xs text-capybara'>{children}</span>
```
선언된 색상이라 이렇게 쓰는게 보기 더 좋을 거 같습니다